### ISSUE TITLE
Run tc.image_classifier.create failed

### ISSUE THREAD

**Issue by fredwzp** [apple/turicreate#1245]
Hi ,everybody,

Recently I have been exploring the image classifier library.
After several weeks success ,unluckily I have been experiencing errors in the function of tc.image_classifier.creat.
This is my python version：Python 2.7.15

Run the code:
`model = tc.image_classifier.create(train_data, target='label')`

then it return to me:
```
---------------------------------------------------------------------------
ToolkitError                              Traceback (most recent call last)
<ipython-input-13-0b735449dfb3> in <module>()
----> 1 model = tc.image_classifier.create(train_data, target='label')

/anaconda2/lib/python2.7/site-packages/turicreate/toolkits/image_classifier/image_classifier.pyc in create(dataset, target, feature, model, validation_set, max_iterations, verbose, seed, batch_size)
    145         feature = _tkutl._find_only_image_column(dataset)
    146 
--> 147     feature_extractor = _image_feature_extractor._create_feature_extractor(model)
    148 
    149     # Extract features

/anaconda2/lib/python2.7/site-packages/turicreate/toolkits/_image_feature_extractor.pyc in _create_feature_extractor(model_name)
     36 
     37     result = extensions.__dict__["image_deep_feature_extractor"]()
---> 38     result.init_options({'model_name': model_name, 'download_path': download_path})
     39     return result
     40 

/anaconda2/lib/python2.7/site-packages/turicreate/extensions.pyc in <lambda>(*args, **kwargs)
    290         elif name in self._functions:
    291             # is it a function?
--> 292             ret = lambda *args, **kwargs: self.__run_class_function(name, args, kwargs)
    293             ret.__doc__ = "Name: " + name + "\nParameters: " + str(self._functions[name]) + "\n"
    294             try:

/anaconda2/lib/python2.7/site-packages/turicreate/extensions.pyc in __run_class_function(self, fnname, args, kwargs)
    277         except RuntimeError as exc:
    278             # Expose C++ exceptions using ToolkitError.
--> 279             raise _ToolkitError(exc)
    280         ret = _wrap_function_return(ret)
    281         return ret

ToolkitError: Unknown exception
```
I tried to download the resnet-50-0000.params and resnet-50-symbol.json manually.But actually it didn't work.
I would appreciate if you have any possible solution.

Thanks,
Fred

**Comment by TobyRoseman** [comment]
Hi @fredwzp -

I'm not really sure what's going on here. Was there any other output?

Looks like you can't download the pretrained models that it needs. You could try deleting the model cache. You can get the model cache path by running:
`tc.config.get_runtime_config()['TURI_CACHE_FILE_LOCATIONS']`

Also make sure you have write permissions to that directory. If you don't have write permission, you can change the location of the model cache by running: `tc.config.set_runtime_config('TURI_CACHE_FILE_LOCATIONS', <new path>)`

**Comment by fredwzp** [comment]
@TobyRoseman 
Thanks for your help.
Yes ,the problem is eliminated now ,and I can run tc.classifier.. smoothly!
Luckily ,I turned on the computer and re-run the code this morning ,then it worked.

In answer to your question, no, there is no other output.

I tried your method and found the model cache path.As I said that I tried to re-download the model cache manually ,but I put them in the picture data path.Actually ,it's the wrong path.What your said is right!
You have proposed a valid method to figure out the possible future problems.

Thanks again,
Fred
